calcom/cal.com
# feat: Oura Ring app

## What does this PR do?

Implementing Oura Ring app. Here is the description:

> Rely on your Oura Ring Readiness score to allow bookings on the days you have data loaded from your ring. A score equal to or greater than 70, means you've recovered well enough so bookings are not blocked, below that, you need to pay attention as you're not fully recovered, so bookings will be blocked. No data means bookings are not blocked.

<kbd><img src="https://github.com/calcom/cal.com/assets/467258/c9745a21-1b00-4b08-9ff3-5278f22f48ae" /></kbd>

## Type of change

- New feature (non-breaking change which adds functionality)

## How should this be tested?

Install the app following the instructions to create a client id and secret and expect availability to be impacted by readiness below 70.

## Mandatory Tasks
